lms/templates/wiki/includes/editor_widget.html

```diff
@@ -1,5 +1,6 @@
 {% load i18n %}
 <textarea {{ attrs }}>{{ content }}</textarea>
+<input aria-describedby="hint_id_content" id="id_content" name="content" type="text">
```

Comment:
did you mean to add a new text input?  I think we want this help text associated with the textarea.

Reply:
Ah, was working off example. Will fix